### PR TITLE
drone 1.x syntax

### PR DESCRIPTION
**Pull request by drpebcak**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,59 +1,97 @@
...
+  - dapper build
+  environment:
+    CROSS: 1
+  privileged: true
+  volumes:
+  - name: socket
+    path: /var/run/docker.sock
+  when:
+    branch:
+    - master
```
Comment (cloudnautique) [inline .drone.yml]: this doesn't work any more, you'll have to follow the rancher/rancher .drone.yml with refs.

File: .drone.yml [hunk 2]
```diff
@@ -1,59 +1,97 @@
...
+  settings:
+    checksum:
+    - sha256
     files:
-     - build/bin/rke_*
-     - dist/artifacts/${DRONE_TAG}/*
+    - "build/bin/rke_*"
+    - "dist/artifacts/${DRONE_TAG}/*"
+    prerelease: true
+  environment:
```
Comment (cloudnautique) [inline .drone.yml]: This has moved to a setting api_key... rancher/rancher for example.

File: .drone.yml [hunk 3]
```diff
@@ -1,59 +1,97 @@
...
-    when:
-      branch: [ master, release/* ]
-      event: tag
-      ref:
-        exclude: [ refs/tags/*rc* ]
+    - sha256
+    files:
+    - "build/bin/rke_*"
+    - "dist/artifacts/${DRONE_TAG}/*"
+  environment:
```
Comment (cloudnautique) [inline .drone.yml]: setting

**Comment by drpebcak** [comment]
@cloudnautique my latest push should resolve all your comments.

**Comment by cloudnautique** [comment]
LGTM

**Pull request merged**